src/ft2font_wrapper.cpp

```diff
@@ -1066,15 +1066,10 @@ static PyObject *PyFT2Font_get_name_index(PyFT2Font *self, PyObject *args, PyObj
 {
     char *glyphname;
```

Comment:
No `const` here in this file?

Reply:
FT_Get_Name_Index takes a non-const char\* as argument (so we'd need a const_cast... not really worth it IMO? can add it if you prefer), even though it does not actually modify it.  See also thread starting at http://lists.nongnu.org/archive/html/freetype/2019-04/msg00004.html.